src/box/tuple.h

```diff
@@ -1015,12 +1011,6 @@ tuple_field_u32(struct tuple *tuple, uint32_t fieldno, uint32_t *out)
 	if (field == NULL)
 		return -1;
 	*out = mp_decode_uint(&field);
-	if (*out > UINT32_MAX) {
```

Comment:
Also looks like a bug - the value is silently truncated to 32 bits if `mp_decode_uint` returns something > `UINT32_MAX`.